.evergreen/auth_oidc/oidc_write_orchestration.py

```diff
@@ -68,9 +69,19 @@ def main():
         "audience": DEFAULT_CLIENT,
         "authorizationClaim": "foo",
         "requestScopes": ["fizz", "buzz"],
-
+        "matchPattern": "test_user1"
+    }
+    provider2_info = {
```

Comment:
Same here. Does generalizing to support an arbitrary number of providers make sense?

Reply:
I don't have plans to expand this, no.